search-enterprise/latest/config/index.md

```diff
@@ -384,3 +384,45 @@ services:
     volumes:
       - ./exactTermSearch.properties:/usr/local/tomcat/webapps/alfresco/WEB-INF/classes/alfresco/search/elasticsearch/config/exactTermSearch.properties
 ```
+
+## Support for different databases
+
+PostgreSQL is the default database for Elasticsearch. You can use different databases with Elasticsearch, but they must be configured within your system and must match the database used by Content Services. The other types of databses supported by Elasticsearch are: MySQL, MariaDB, Microsoft Sql Server, and Oracle.
```

Comment:
I think it would be better to use the _Search Enterprise_ instead of the _Elasticsearch_ in this paragraph. The _Elasticsearch_ is a separate product and it doesn't require a database.

Reply:
Good point.